src/sentry/integrations/slack/event_endpoint.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import absolute_import
+
+import logging
+import json
+import re
+import six
+
+from six.moves.urllib.parse import parse_qs, urlencode, urlparse, urlunparse
+
+from sentry import http, options
+from sentry.api.base import Endpoint
+from sentry.models import Group, Integration, Project
+
+logger = logging.getLogger('sentry.integrations.cloudflare')
+
+_link_regexp = re.compile(r'^https?\://[^/]+/[^/]+/[^/]+/issues/(\d+)/')
```

Comment:
Surely that regex catches too much? Eg: it would catch github urls as well.

Reply:
It’s intended to. Slack only let’s you unfurl urls for your domains so this seems safe.